pkg/mediationcontainer/client_websocket_transport.go

```diff
@@ -212,16 +230,19 @@ func (clientTransport *ClientWebSocketTransport) performWebSocketConnection() er
 	glog.Infof("[performWebSocketConnection]: %s", vmtServerUrl)
 
 	for !clientTransport.closeRequested { // only set when CloseTransportPoint() is called
```

Comment:
When remote mediation client is stopped explicitly, we want to close transport layer. clientTransport.closeRequested is set at [line 104](https://github.com/turbonomic/turbo-go-sdk/pull/18/files/2fb80c07f16e867a31666bbac18d130387d72e70#diff-ab2326914686a36993657f52e15b6884L97).

I think here we want to stop the connection explicitly, otherwise the whole for loop wont stop until the connection has been successful built.